### PR TITLE
Tajaran Tackling

### PR DESCRIPTION
**Pull request by RadiantFlash7**
## About The Pull Request
Grants Tajarans a tackling bonus like moths.

## Why It's Good For The Game
Tajarans currently get unique eyes, which are incredibly weak to flashes and welding, in exchange for minorish night vision capabilities.
However, they were also given unique sensitive ears which will pretty much always be deafened from flashbangs. Theres no real upside to this- so I figure making Tajaran ears apply a bonus to tackling would be a good idea. 
Plus, cat pouncing memes is pretty great.

## Proof Of Testing
It Just Works ™️ 
But in seriousness, I tackled a few mobs on a test server. The change seems to be working as intended.

## Changelog

:cl:
balance: gave tajarans better tackles to compensate for weak ears
/:cl:

### CODE REVIEW

**Comment by ReturnToZender** [comment]
I'm cool with anthro catboys being the new tackle powergame race

**Comment by RadiantFlash7** [comment]
> I'm cool with anthro catboys being the new tackle powergame race
> 
> This isn't sarcasm, this is unironic

Given how hard they got gimped with their eyes and ears changes, I'm kind of glad to hear it. The partial NV from the eyes is practically effectively on par with the NV trait you can pick, and you get so many negatives in return.

**Comment by ReturnToZender** [comment]
> > I'm cool with anthro catboys being the new tackle powergame race
> > This isn't sarcasm, this is unironic
> 
> Given how hard they got gimped with their eyes and ears changes, I'm kind of glad to hear it. The partial NV from the eyes is effectively on par with the NV trait you can pick, and you get so many negatives in return.

It has nothing to do with their balance I just think Tajarans are the ideal form of catboy/catgirl and I will die on that hill, @Tajarans call me so we can set up a time to ERP

**Comment by RadiantFlash7** [comment]
> > > I'm cool with anthro catboys being the new tackle powergame race
> > > This isn't sarcasm, this is unironic
> > 
> > 
> > Given how hard they got gimped with their eyes and ears changes, I'm kind of glad to hear it. The partial NV from the eyes is effectively on par with the NV trait you can pick, and you get so many negatives in return.
> 
> It has nothing to do with their balance I just think Tajarans are the ideal form of catboy/catgirl and I will die on that hill, @Tajarans call me so we can set up a time to ERP

Mood.

**Comment by projectkepler-ru** [comment]
we should have this on skyrat

**Comment by Erol509** [comment]
Please add a toggle for this, I dont want to get my head smashed when trying to throw an item.

**Comment by projectkepler-ru** [comment]
> Please add a toggle for this, I dont want to get my head smashed when trying to throw an item.

you completley misunderstood what this does

**Comment by RadiantFlash7** [comment]
> we should have this on skyrat

As far as I'm aware, the tajaran changes to their eyes and ears was a bubberstation change. Not sure you'd want this to skyrat without that as well

**Comment by RadiantFlash7** [comment]
> > > we should have this on skyrat
> 
> > 
> 
> > As far as I'm aware, the tajaran changes to their eyes and ears was a bubberstation change. Not sure you'd want this to skyrat without that as well
> 
> 
> 
> we can afford to do that :P

You might have a riot on your hands from taj players. Hypersensitive eyes and ears is such a pain to play around. Either way, tajaran being better at pouncing is still funny

**Comment by pixelkitty286** [comment]
> > > > we should have this on skyrat
> > 
> > 
> > > 
> > 
> > 
> > > As far as I'm aware, the tajaran changes to their eyes and ears was a bubberstation change. Not sure you'd want this to skyrat without that as well
> > 
> > 
> > we can afford to do that :P
> 
> You might have a riot on your hands from taj players. Hypersensitive eyes and ears is such a pain to play around. Either way, tajaran being better at pouncing is still funny

me removing the hypersensitive flag after seeing this and making it only sensitive (ears were done because if felinids have it why don't actual cat people have it.)

**Comment by projectkepler-ru** [comment]
> > > > > > we should have this on skyrat
> > > > 
> > > > 
> > > > > 
> > > > 
> > > > 
> > > > > As far as I'm aware, the tajaran changes to their eyes and ears was a bubberstation change. Not sure you'd want this to skyrat without that as well
> > > > 
> > > > 
> > > > we can afford to do that :P
> > > 
> > > 
> > > You might have a riot on your hands from taj players. Hypersensitive eyes and ears is such a pain to play around. Either way, tajaran being better at pouncing is still funny
> > 
> > 
> > me removing the hypersensitive flag after seeing this and making it only sensitive (ears were done because if felinids have it why don't actual cat people have it.)
> 
> Wait really? Awh heck, does this mean My catboys won’t get s pouncing bonus?
> 
> Also felinids can lick their injuries to help stop bleeding, which is a feature tajs didn't get if I recall.

they can get it but I'd prferably push this up to skyrat too

**Comment by pixelkitty286** [comment]
> > > > > > > we should have this on skyrat
> > > > > 
> > > > > 
> > > > > > 
> > > > > 
> > > > > 
> > > > > > As far as I'm aware, the tajaran changes to their eyes and ears was a bubberstation change. Not sure you'd want this to skyrat without that as well
> > > > > 
> > > > > 
> > > > > we can afford to do that :P
> > > > 
> > > > 
> > > > You might have a riot on your hands from taj players. Hypersensitive eyes and ears is such a pain to play around. Either way, tajaran being better at pouncing is still funny
> > > 
> > > 
> > > me removing the hypersensitive flag after seeing this and making it only sensitive (ears were done because if felinids have it why don't actual cat people have it.)
> > 
> > 
> > Wait really? Awh heck, does this mean My catboys won’t get s pouncing bonus?
> > Also felinids can lick their injuries to help stop bleeding, which is a feature tajs didn't get if I recall.
> 
> they can get it but I'd prferably push this up to skyrat too

tajaran changes are only on here because I don't really care for dealing with SR

**Comment by pixelkitty286** [comment]
> > > > > > we should have this on skyrat
> > > > 
> > > > 
> > > > > 
> > > > 
> > > > 
> > > > > As far as I'm aware, the tajaran changes to their eyes and ears was a bubberstation change. Not sure you'd want this to skyrat without that as well
> > > > 
> > > > 
> > > > we can afford to do that :P
> > > 
> > > 
> > > You might have a riot on your hands from taj players. Hypersensitive eyes and ears is such a pain to play around. Either way, tajaran being better at pouncing is still funny
> > 
> > 
> > me removing the hypersensitive flag after seeing this and making it only sensitive (ears were done because if felinids have it why don't actual cat people have it.)
> 
> Wait really? Awh heck, does this mean My catboys won’t get s pouncing bonus?
> 
> Also felinids can lick their injuries to help stop bleeding, which is a feature tajs didn't get if I recall.

see https://github.com/Bubberstation/Bubberstation/pull/962 then https://github.com/Bubberstation/Bubberstation/pull/1530 if you want to know what I was talking about

File: code/datums/components/tackle.dm [hunk 1]
```diff
@@ -436,6 +436,9 @@
 	var/obj/item/organ/external/wings/sacker_wing = sacker.get_organ_slot(ORGAN_SLOT_EXTERNAL_WINGS)
 	if(sacker_wing)
 		attack_mod += 2
+	var/obj/item/organ/internal/ears/cat/tajaran/sacker_tajaran_ears = sacker.get_organ_slot(ORGAN_SLOT_EARS)//Bubberstation Edit start- Cat's are particularly good at pouncing
+	if(HAS_TRAIT(sacker, TRAIT_CATLIKE_GRACE) && sacker_tajaran_ears)
+		attack_mod += 2//Bubberstation Edit finish
```
Comment (Majkl-J) [inline code/datums/components/tackle.dm:441]: You're more likely to lack the ears than the trait. Also needs to typecheck if you want it tied to the organ type  specifically.

```suggestion
	var/obj/item/organ/internal/ears/cat/tajaran/sacker_tajaran_ears = sacker.get_organ_slot(ORGAN_SLOT_EARS)//Bubberstation Edit start- Cat's are particularly good at pouncing
	if(istype(sacker_tajaran_ears) && HAS_TRAIT(sacker, TRAIT_CATLIKE_GRACE))
		attack_mod += 2//Bubberstation Edit finish
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]

File: code/datums/components/tackle.dm (hunk 1 above)
Comment (RadiantFlash7) [reply code/datums/components/tackle.dm]: Yeah, Fair point. Thank you.

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**